dolphin/processor/recipe.py

```diff
@@ -570,7 +570,6 @@ def fix_params(self, model_component, index=None):
                 param_list_with_index.append([i, param_list])
 
         key = '{}_add_fixed'.format(model_component)
```

Comment:
Why did you remove the empty line before the `return` statement here?

Reply:
Oops. To find the bug in the code, I added a lot of print functions to look at its output. Once I fix the bug, I removed the print functions and  accidentally delete the empty line